packages/terra-section-header/src/SectionHeader.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Arrange from 'terra-arrange';
+import styles from './SectionHeader.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Text to be displayed on the SectionHeader.
+   */
+  title: PropTypes.node.isRequired,
+  /**
+   * Callback function triggered when the accordion icon is clicked.
+   */
+  onClick: PropTypes.func,
+  /**
+   * Whether the accordion icon should be displayed in its 'open' or 'closed' position.
+   */
+  isOpen: PropTypes.bool,
+};
+
+const defaultProps = {
+  onClick: undefined,
+  isOpen: false,
+};
+
+const SectionHeader = ({
+  title,
+  onClick,
+  isOpen,
+  ...customProps
+}) => {
+  if ((process.env.NODE_ENV !== 'production') && (!onClick && isOpen)) {
+    // eslint-disable-next-line no-console
+    console.warn('\'isOpen\' are intended to be used only when \'onClick\' is provided.');
+  }
+
+  const attributes = Object.assign({}, customProps);
+
+  if (onClick) {
+    // makes the div tabable
+    attributes.tabIndex = '0';
+  }
+
+  const iconClassNames = cx([
+    'accordion-icon',
+    { 'is-open': onClick && isOpen },
+  ]);
+
+  const accordionIcon = (
+    <div className={cx('accordion-icon-wrapper')}>
+      <span className={iconClassNames} />
+    </div>
+  );
+
+  const sectionHeaderClassNames = cx([
+    'section-header',
+    { 'is-interactable': onClick },
+    customProps.className,
+  ]);
+
+  // allows us to set an onClick on the div
+  /* eslint-disable jsx-a11y/no-static-element-interactions */
+  return (
+    <div {...attributes} onClick={onClick} className={sectionHeaderClassNames}>
+      <Arrange
+        fitStart={onClick && accordionIcon}
+        fill={<span className={cx('title')}>{title}</span>}
```

Comment:
If it's intended to the the title of this header, can we use an h1, h2, h3, h4, etc.. instead?

Reply:
Done 40f4a3a67311c055d6b8ab9d9e519e185976ad3a

Reply:
Is h2 what Chris/Neil wants?

Reply:
Reconsidering - a section header might be used as any level, not always as a guaranteed h2 position, so safest bet is to make it additional level prop so consumer can create correct heading element needed for appropriate screenreader accessibility by position within DOM. 

Proposal to amend Tech Design to add:

Prop Name | Type | Is Required | Default Value | Description | 
------------ | ------------ | ------------ | ------------ | ------------ |
`level` | number | optional |  **2** |  Sets the heading level. Sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`. |

To follow terra-heading:
https://github.com/cerner/terra-core/blob/467c0920fdec01b138c49c0e9a8874db7485d3a1/packages/terra-heading/src/Heading.jsx#L17

Reply:
Done 496797fb2b84772f486c938cb2bc06a29c76c67f
f8a54129eb3ae9a3461727fbd58b7f37490c2ca8